WooCommerce/Classes/Model/Address+Woo.swift

```diff
@@ -86,6 +86,15 @@ extension Address {
         self == .empty
     }
 
+    /// Erases the address components that are also part of a Tax Rate. Call this if you want to unlink an address from a tax rate.
+    /// 
+    func resettingTaxRateComponents() -> Address {
```

Comment:
The same here, should we use "reset"? 
```suggestion
    func resetTaxRateComponents() -> Address {
```

Reply:
Same as above